protobuf/README.md

```diff
@@ -8,12 +8,12 @@ Check out the [Flow Access API specification](/docs/content/access-api-spec.md).
 
 ## Generating Go stubs
 
-You can use [prototool](https://github.com/uber/prototool) to generate gRPC client stubs in a variety of languages.
+You can use [buf](https://github.com/bufbuild/buf) to generate gRPC client stubs in a variety of languages.
 
-Running the command below (in the current directory) will generate stubs for Go:
+Running the command below (in the root directory) will generate stubs for Go:
```

Comment:
so the `flow` repo has a lot more than the protobuf files. It also has all the documentation under `content` and the node operator list under `nodeoperators`. Hence, it would be best if we can keep the `makefile` and other related things to generate the protobuf go code under the `protobuf` directory instead of outside.

Reply:
Addressed comment in 5c12bdd